adapters/medianet/medianet.go

```diff
@@ -0,0 +1,139 @@
+package medianet
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/mxmCherry/openrtb/v15/openrtb2"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/errortypes"
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+type MedianetAdapter struct {
+	endpoint string
+}
+
+func (a *MedianetAdapter) MakeRequests(request *openrtb2.BidRequest, reqInfo *adapters.ExtraRequestInfo) ([]*adapters.RequestData, []error) {
+	var errs []error
+
+	if len(request.Imp) == 0 {
+		return nil, []error{&errortypes.BadInput{
```

Comment:
Prebid server core filters out requests with empty []Imp arrays. In production line 23 would be unreachable.

Reply:
Resolved !!

Reply:
Thank you